templates/taskcat-ci-entrypoint.template.yaml

```diff
@@ -0,0 +1,506 @@
+      Description: Merge GitHub branches
+      Handler: index.handler
+      Role: !GetAtt GitMergeRole.Arn
+      Runtime: python3.9
+      Timeout: 30
+  CodeBuildServiceRole:
+    Type: AWS::IAM::Role
+    Properties:
+      Path: /
+      AssumeRolePolicyDocument:
+        Version: 2012-10-17
+        Statement:
+          - Effect: Allow
+            Action: sts:AssumeRole
+            Principal:
+              Service: codebuild.amazonaws.com
+      ManagedPolicyArns:
+        - !Sub arn:${AWS::Partition}:iam::aws:policy/AdministratorAccess
+  CodeBuild:
+    Type: AWS::CodeBuild::Project
+    Properties:
+      Description: !Sub >-
+        Submit build jobs for ${GitHubRepoName} as part of a CI/CD pipeline.
+      ServiceRole: !GetAtt CodeBuildServiceRole.Arn
+      Artifacts:
+        Type: CODEPIPELINE
+      Environment:
+        Type: LINUX_CONTAINER
+        ComputeType: BUILD_GENERAL1_SMALL
+        Image: aws/codebuild/standard:5.0
+        EnvironmentVariables:
+          - Name: ARTIFACT_BUCKET
+            Value: !Ref ArtifactBucket
+          - Name: PROJECT_NAME
+            Value: !Ref GitHubRepoName
+          - Name: PROJECT_OWNER
+            Value: !Ref GitHubOwner
+          - Name: SOURCE_BRANCH
+            Value: !Ref SourceBranch
+        PrivilegedMode: true
+      Visibility: PRIVATE
+      Source:
+        Type: CODEPIPELINE
+        GitCloneDepth: 1
+        BuildSpec: !Sub |
+          version: 0.2
+
+          env:
+            shell: bash
+            secrets-manager:
+              GITHUB_TOKEN: ${Secrets}:Token
+              GITHUB_USERNAME: ${Secrets}:Username
+              TASKCAT_OVERRIDES: arn:${AWS::Partition}:secretsmanager:${AWS::Region}:${AWS::AccountId}:secret:${GitHubRepoName}
+
+          phases:
+            install:
+              runtime-versions:
+                python: 3.x
+              commands:
+                - echo "Entered the install phase..."
+                - apt-get -qy update
+                - echo "Installing system dependencies..."
+                - apt-get -qy install zip gzip tar
+                - echo "Installing python dependencies..."
+                - pip3 install cfn-lint taskcat yq
+                - echo "Configuring taskcat..."
+                - |
+                  cat << EOF > ~/.taskcat.yml
+                  general:
+                    s3_regional_buckets: true
+                  EOF
+                - echo "Installing custom cfn-lint rules..."
+                - wget -nv https://github.com/aws-quickstart/qs-cfn-lint-rules/archive/refs/heads/main.zip
+                - unzip -q ./main.zip
+                - mv ./qs-cfn-lint-rules-main ~/qs-cfn-lint-rules
+                - pip3 install -e ~/qs-cfn-lint-rules
+                - |
+                  cat << EOF > ~/.cfnlintrc
+                  append_rules:
+                    - ~/qs-cfn-lint-rules/qs_cfn_lint_rules/
+                  EOF
```

Comment:
Please use [the pre-commit hook](https://github.com/aws-quickstart/qs-cfn-lint-rules/blob/main/.pre-commit-hooks.yaml#L6).

Reply:
Implemented in f17f9c1717acac9927d2c02b1845443ea0647b82.